datadog_checks_base/datadog_checks/base/checks/openmetrics/v2/scraper.py

```diff
@@ -361,11 +362,17 @@ def stream_connection_lines(self):
         Yield the connection line.
         """
 
-        with self.get_connection() as connection:
-            # Media type will be used to select parser dynamically
-            self._content_type = connection.headers.get('Content-Type', '')
-            for line in connection.iter_lines(decode_unicode=True):
-                yield line
+        try:
+            with self.get_connection() as connection:
+                # Media type will be used to select parser dynamically
+                self._content_type = connection.headers.get('Content-Type', '')
+                for line in connection.iter_lines(decode_unicode=True):
+                    yield line
+        except ConnectionError as e:
+            if self.ignore_connection_errors:
+                self.warning("OpenMetrics endpoint is not accessible")
```

Comment:
```suggestion
                self.log.warning("OpenMetrics endpoint is not accessible. Will not raise exception since `ignore_connection_errors` set to true")
```
`self.warning` will output a warning in the results rather than just log the warning.